mason/breeders_toolbox/trial/trial_details.mas

```diff
@@ -24,6 +24,9 @@ $trial_owner
 
 </%args>
 
+<& /breeders_toolbox/folder/folder_new.mas, project_type=>'field_trial', breeding_program_name=>$breeding_program_name, breeding_program_id=>$breeding_program_id &>
+<& /breeders_toolbox/folder/folder_set.mas, project_type => 'field_trial', breeding_program_id => $breeding_program_id, breeding_program_name =>$breeding_program_name, trial_id => $trial_id, trial_name =>$trial_name &>
+
```

Comment:
these two mason files are already loaded in trial.mas so you get DOM errors for duplicated IDs